examples/09_Kotlin_JS/02_js_function.md

```diff
@@ -19,6 +32,6 @@ fun main(){
 
 </div>
 
-1. Create a JavaScript object literal. The `js(...)` function return type is `dynamic`.
```

Comment:
I would leave it as Create as it is an instruction, or use the passive if it is a description so Creates